sonic_y_cable/y_cable.py

```diff
@@ -1237,3 +1262,90 @@ def get_nic_voltage_temp(physical_port):
         return -1
 
     return temp, voltage
+
+def get_local_temperature(physical_port):
+
+    curr_offset = OFFSET_INTERNAL_TEMPERATURE
+    if platform_chassis is not None:
+        result = platform_chassis.get_sfp(physical_port).read_eeprom(curr_offset, 1)
+        if y_cable_validate_read_data(result, 1, physical_port, "local temperature") == EEPROM_READ_DATA_INVALID:
+            return EEPROM_ERROR
+        temp = result[0]
+    else:
+        helper_logger.log_error("platform_chassis is not loaded, failed to get local temp")
+        return -1
+
+    return temp
+
```

Comment:
Two blank lines between top-level functions per PEP8. Please do this throughout the file.

Reply:
fixed

Reply:
fixed